app/views/idv/hybrid_handoff/show.html.erb

```diff
@@ -10,7 +10,11 @@
 <% end %>
 
 <%= render PageHeadingComponent.new do %>
-  <%= t('doc_auth.headings.hybrid_handoff') %>
+  <% if IdentityConfig.store.idv_phone_question_a_b_testing[:show_phone_question] == 1 %>
```

Comment:
the flag to determine which variation of hybrid handoff page will not be available in `IdentityConfig.store.idv_phone_question_a_b_testing[:show_phone_question]`. After we merge #9315, we will be able to add a global variable to HybridHandoffController#show to determine which variation of the hybrid handoff page to display

Reply:
Thank you for bringing that to my attention! I will wait for #9315 to be merged and then rebase and fix this.

Reply:
Now that I have rebased, I was able to make the change and use a variable in [15d674f](https://github.com/18F/identity-idp/pull/9316/commits/15d674f0f9124823680545ba6034b3c60f8ddf14).